src/lib/work_selector/work_lib.ml

```diff
@@ -159,4 +159,40 @@ module Make (Inputs : Intf.Inputs_intf) = struct
         all_pending_work ~snark_pool all_todo_statements
     | Some fee ->
         expensive_work all_todo_statements ~fee
+
+  (* Seen/Unseen transfer jobs that are not in the snark pool yet *)
+  let pending_user_work_statements ~snark_pool ~fee_opt (state : State.t) =
+    let work_filter (witness : Inputs.Transaction_witness.t option) =
+      match witness with
+      | None ->
+          false
+      | Some witness ->
+          Inputs.Transaction.is_user_transaction
```

Comment:
I'm not sure I understand the goal here. All of the snark work needs to be done (and ideally thus should be available from the GraphQL command)

Reply:
I thought the goal was just too eventually filter queries based on certain characteristics i.e fetch snark work like <insert traits here>. I saw this change as poc for this concept with just signed commands. I don’t think the intention is to replace functionality for querying all work.